### PR TITLE
[PULL REQUEST] SQL Database Build #57

### PR DESCRIPTION
**Pull request by nmad-sandag**
**Describe this pull request. What changes are being made?**
The Cohort Component model currently only outputs csv files for each run to an [output folder]. The results from each run is saved in SQL database for tracking, or reporting across hundreds of model runs nor for scenario/run tracking

**What issues does this pull request address?**
#57 closed

### CODE REVIEW

File: sql/db_build/table_creation.sql
Comment (GregorSchroeder) [inline sql/db_build/table_creation.sql:1]: - use brackets `[ ]` for all column names, table names, schema names
- suggest renaming `[metadata].[ccm_run]` to just `[metadata].[run]`
- suggest renaming output tables to match their default output csv file names
  - [outputs].[components]
  - [outputs].[population]
  - [outputs].[rates]
- add foreign key constraints in all tables back to the `[metadata].[run]` table
- remove `[inputs]` schema creation as it is not used
- add `CLUSTERED COLUMNSTORE INDEX`s to all `[output]` tables and `UNIQUE` constraints on their unique composite keys

File: python/etl.py
Comment (GregorSchroeder) [inline python/etl.py:1]: - remove all ETL related things as no issue has been created
- this looks overly complex for what should be a generic read of csv files and insert into sql tables verbatim for almost everything

**Review by GregorSchroeder — CHANGES_REQUESTED** [review]

File: python/etl.py
Comment (nmad-sandag) [reply python/etl.py:1]: I will work on making it less complex :)

**Review by GregorSchroeder — APPROVED** [review]
Looks good! Please make a new issue describing the new ETL feature wanted and then begin work on that.

**Pull request merged**